src/GRA.Controllers/MissionControl/TriggersController.cs

```diff
@@ -435,8 +446,17 @@ public async Task<IActionResult> Edit(int id)
                 BadgeRequiredList = string.Join(",", trigger.BadgeIds),
                 ChallengeRequiredList = string.Join(",", trigger.ChallengeIds),
                 SystemList = new SelectList(await _siteService.GetSystemList(), "Id", "Name"),
-                ProgramList = new SelectList(await _siteService.GetProgramList(), "Id", "Name")
-            };
+                ProgramList = new SelectList(await _siteService.GetProgramList(), "Id", "Name"),
+                IgnorePointLimits = UserHasPermission(Permission.IgnorePointLimits)
+        };
+            if (site.MaxPointsPerTrigger.HasValue)
+            {
+                viewModel.MaxPointLimit = site.MaxPointsPerTrigger.Value;
+            }
+            if (site.MaxPointsPerTrigger.HasValue && trigger.AwardPoints > site.MaxPointsPerTrigger)
+            {
+                ShowAlertWarning("This Trigger's number of points awarded is higher than what is allowed by the configuration.");
+            }
```

Comment:
These can be combined into nested if statements.

Reply:
Fixed in commit c2ef27e7367824dd9469c5a2ab675a793f794448